node/hack/query/utils/fetchCurrentGuardianSet.go

```diff
@@ -67,6 +69,7 @@ func FetchCurrentGuardianSet(network common.Environment) (uint32, *ethAbi.Struct
 	case common.UnsafeDevNet:
 	case common.GoTest:
 		ethContract = "0xC89Ce4735882C9F0f0FE26686c53074E09B0D550"
+	case common.AccountantMock:
```

Comment:
Just want to check, what's the intended action on this match?

Reply:
It should never happen, so treating it the same as the default (returning an error) is reasonable.